libs/api/domains/application/src/lib/application.resolver.ts

```diff
@@ -177,4 +179,12 @@ export class ApplicationResolver {
   ): Promise<PresignedUrlResponse> {
     return this.applicationService.presignedUrl(input, user)
   }
+
+  @Query(() => FileContentAsBase64Response, { nullable: true })
+  getFileContentAsBase64(
+    @Args('input') input: FileContentAsBase64Input,
+    @CurrentUser() user: User,
+  ): Promise<FileContentAsBase64Response> {
+    return this.applicationService.getFileContentBase64(input, user)
+  }
```

Comment:
It's not great to transfer big files as base64 through our NodeJS services, as it can consume a lot of memory. Especially in our GraphQL API which is handling a lot of stuff.

When we've figured out the security side of this functionality, we should consider changing this to either return a presigned S3 url, or consider using the download service to stream the file directly.